jdaviz/configs/imviz/plugins/tools.py

```diff
@@ -102,17 +102,17 @@ class BlinkOnce(CheckableTool):
     icon = os.path.join(ICON_DIR, 'blink.svg')
     tool_id = 'jdaviz:blinkonce'
     action_text = 'Go to next image'
-    tool_tip = ('Click on the viewer to display the next image, '
-                'or you can also press the "b" key anytime')
+    tool_tip = ('Click on the viewer or press "b" to display the next image, '
+                'or right-click or press "B" to display the previous')
 
     def activate(self):
-        self.viewer.add_event_callback(self.on_click, events=['click'])
+        self.viewer.add_event_callback(self.on_click, events=['click', 'contextmenu'])
 
     def deactivate(self):
         self.viewer.remove_event_callback(self.on_click)
 
     def on_click(self, data):
-        self.viewer.blink_once()
+        self.viewer.blink_once(reversed=data['event']=='contextmenu')  # noqa: E225
```

Comment:
```suggestion
        self.viewer.blink_once(reversed=data.get('event')=='contextmenu')  # noqa: E225
```
would prevent the same type of test failure that was just fixed in the future, but probably isn't necessary because this should only be called directly in the tests.

Reply:
Yeah. Real callback would have event in it always, so I don't think this is necessary. It was the test that was buggy. Thanks anyway!